### PR TITLE
New UI has been created with nav bar and cards

### PR DESCRIPTION
**Pull request by sonihiten5**

### CODE REVIEW

**Comment by anku580** [comment]
@sonihiten5 Can you please share a screenshot and resolve the merge conflicts as well?

**Comment by sonihiten5** [comment]
should i send you UI screen shot or code

**Comment by anku580** [comment]
@sonihiten5 Yes, please attach the UI screenshot.

**Comment by sonihiten5** [comment]
Screenshot 2020-10-02 000629

**Comment by sonihiten5** [comment]
i Have also resolve the conflicts with new links of the user by other members also

**Comment by anku580** [comment]
This one looks good :clap: 

Can you please format the code so that it would be easy to maintain and remove extra spaces?

**Comment by sonihiten5** [comment]
Sure

File: index.html [hunk 1]
```diff
@@ -17,51 +18,160 @@
...
-      <li><a href="https://www.linkedin.com/in/daniela-hernandez-149310170/" target="blank">Daniela Hernandez - naekkeoya</a></li>
-      <li><a href="https://www.linkedin.com/in/kishan-takoordyal-99816b180/" target="blank">Kishan Takoordyal - EdgeKing810</a></li>
-	    <li><a href="https://www.linkedin.com/in/muskan-deep-kaur-63b7701a4/" target="blank">Muskan deep Kaur</a></li>
+  
     </ul>
     </div>
   </div>
-  </div>
+
+
```
Comment (anku580) [inline index.html]: Can you remove these extra spaces?

File: index.html [hunk 2]
```diff
@@ -17,51 +18,160 @@
...
+     
+      </div>
+    </div>
+   
     <!-- Linkedin Usernames -->
     <div class="row mt-3">
       <div class="col-md-12">
-        <img src="./linkedin.jpg" height="20px">
-        <span class="linkedin-header">Linkedin Detail</span>
+
```
Comment (anku580) [inline index.html]: Can you remove these extra spaces?

File: index.html [hunk 3]
```diff
@@ -17,51 +18,160 @@
...
+    
+      </div>
+
+
+
+
     </div>
+
+
+
```
Comment (anku580) [inline index.html]: Can you remove these extra spaces?

File: index.html [hunk 4]
```diff
@@ -17,51 +18,160 @@
...
+          </div>
+
+          <div class="d-flex flex-column">
+           
+          </div>
+        </div>
+    
+      </div>
+
+
```
Comment (anku580) [inline index.html]: Can you remove these extra spaces?

File: index.html [hunk 5]
```diff
@@ -17,51 +18,160 @@
...
```
Comment (anku580) [inline index.html:80]: Can you remove these extra spaces?

File: index.html [hunk 6]
```diff
@@ -17,51 +18,160 @@
...
+        </div>
+
+        <div class="row m-auto justify-content-center ">
+          <div class="col-md-6">
+            <ul>
+              <li><a href="https://github.com/anku580" target="blank">Anku</a></li>
+              <li><a href="https://github.com/MohammedAkhil" target="blank">Mohammed Akhil</a></li>
+
+
+              <li><a href="https://github.com/sonihiten5" target="_blank">Hiten Soni</a></li>
```
Comment (anku580) [inline index.html:66]: Please format all the `<li>` tags.

File: index.html [hunk 7]
```diff
@@ -17,51 +18,160 @@
...
+          <button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>
+        </form>
       </div>
-    </div>
-    <div>
+    </nav>
+</header>
+
+
+
```
Comment (anku580) [inline index.html]: Can you remove these extra spaces?

File: index.html [hunk 8]
```diff
@@ -17,51 +18,160 @@
 </head>
 
 <body>
-  <div class="container-fluid">
-    <div class="row nav">
-      <div class="col-md-12">
-        <span class="nav-text">Hacktoberfest 2020</span>
+
+
```
Comment (anku580) [inline index.html]: Can you remove these extra spaces?

**Review by anku580 — CHANGES_REQUESTED** [review]
I've added some comments, please address it.

**Comment by sonihiten5** [comment]
i have removed all the extra spaces

**Comment by sonihiten5** [comment]
Also with conflicts resolved with latest code

**Review by anku580 — APPROVED** [review]
Thank you for contributing.

Can you also raise another PR and attach the screenshot to the README file?

**Comment by sonihiten5** [comment]
Yes i have raised and attached a screenshot in readme

**Pull request merged**